### DISCUSSION TITLE
Add a captions for product images by default

### DISCUSSION THREAD

**Discussion by hrostyk** [PrestaShop/PrestaShop#40169] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

When add a new image to a product, the image caption remains empty

### Describe the solution you'd like

1. When add a new product image, the caption for image copy from product title for all languages.
2. Add button... "Use title as caption" 

### Alternatives you've considered

In most cases, the captions uses the product title. Copy and paste manually... bad experience. The captions (alt and title on FO) are used by search engines. Blank captions have an effect on SEO. It is better to copy the product title than empty.

### Do you plan to work on this feature?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by sallemiines** [comment]
Hello @hrostyk ,

Thank you for your suggestion. The Maintainer Team will take it into consideration for future developments. 

Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you create it. 

If you or anyone else is willing to [contribute this feature](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/) to the PrestaShop Project, the teams will be happy to help by ensuring that it fits well within the project. Please let us know if that is the case!

Thank you
